src/xrCore/xr_cpuid.cpp

```diff
@@ -52,7 +52,11 @@ void nativeCpuId(int regs[4], int i)
 #ifndef XR_PLATFORM_WINDOWS
 #include <thread>
 
+#ifdef XR_PLATFORM_WINDOWS
```

Comment:
Remove that from here, here it is unnecassary.

Reply:
Done